eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -117,8 +119,9 @@ function DeployStressPackage(
     [object]$pkg,
     [string]$deployId,
     [string]$environment,
-    [string]$repository,
-    [boolean]$pushImages
+    [string]$repositoryBase,
+    [boolean]$pushImages,
+    [boolean]$login
```

Comment:
switch instead of bool maybe?

Reply:
FWIW, I was going to comment the same, but there were already so many `[bool]` parameters in the other scripts. IMO, `[switch]` is idiomatic, but I really only care about places like the _New-TestResources.ps1_ script that are meant to be callable by humans. Changing all existing and new `[bool]` parameters to `[switch]` feels out of scope of this PR, so perhaps consistency within each script is better.